### ISSUE TITLE
[FR]: keyboard navigation with file, formula and rollup

### ISSUE THREAD

**Issue by imeed166** [RafaelGB/obsidian-db-folder#731]
### Present your request

I'd like to have keyboard navigation (even I can't edit the cells) for these 3 types of columns (`file`, `formula` and `rollup`). Because if these columns are in the middle of the table, I can't navigate all the table with the keyboard, I'm stuck in one part of the table.

### For which platform do you request this request??

Desktop

**Comment by RafaelGB** [comment]
Yeah, I noticed that and it's already fixed. Will be included with the next version =)

Thanks fro the report!
